packages/shared/providers.yaml

```diff
@@ -5578,6 +5578,27 @@ twinfield:
     disable_pkce: true
     docs: https://docs.nango.dev/integrations/all/twinfield
 
+twenty-crm:
+    display_name: Twenty CRM
+    categories:
+        - crm
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://api.twenty.com/rest
+        headers:
+            Authorization: Bearer ${apiKey}
+        retry:
+            after: 'Retry-After'
+        verification:
+            method: GET
+            endpoint: /companies
+    docs: https://docs.nango.dev/integrations/all/twenty-crm
+    credentials:
+        apiKey:
```

Comment:
might be good to include a pattern and example in here.

Reply:
couldn't find any information on rate limiting of Twenty CRM APIs

Reply:
I have removed the duplicate link

Reply:
Some providers may not include this, so you can omit it if not present in the docs. Please check [comment](https://github.com/NangoHQ/nango/pull/2946#discussion_r1830428059)